src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -250,6 +250,9 @@ function MoneyRequestConfirmPage(props) {
                         props.currentUserPersonalDetails.login,
                         props.currentUserPersonalDetails.accountID,
                         trimmedComment,
+                        '',
+                        '',
```

Comment:
These values correspond to the category and tag parameters in [startSplitBill](https://github.com/Expensify/App/blob/3349f6bec1d77a8c4d40e250e180dfc2eb78aa3d/src/libs/actions/IOU.ts#L2775-L2786). We were mistakenly passing the receipt and existingSplitChatReportID instead.